modules/jaggery-apps/app-store-web/themes/store/js/handlebars.js

```diff
@@ -117,6 +117,51 @@ Handlebars.registerHelper('with', function(context, options) {
 Handlebars.registerHelper('log', function(context) {
   Handlebars.log(context);
 });
+
```

Comment:
This handlebar will called by the hbs files. Putting hbs cade comment in a js file gives kind of an ambiguity meaning.